sdk/src/main/scala/com/horizen/account/state/AccountState.scala

```diff
@@ -223,13 +227,30 @@ class AccountState(val params: NetworkParams,
 
   override def getConsensusEpochNumber: Option[ConsensusEpochNumber] = getView.getConsensusEpochNumber
 
+  def getOrderedForgingStakesInfoSeq() : Seq[ForgingStakeInfo] = {
+    val stateView: AccountStateView = getView
+    stateView.getOrderedForgingStakeInfoSeq()
+  }
+
   // Returns lastBlockInEpoch and ConsensusEpochInfo for that epoch
   // Identical to the SidechainState.getCurrentConsensusEpochInfo method
   def getConsensusEpochInfo: (ModifierId, ConsensusEpochInfo) = {
```

Comment:
need to check if this implementation is exactly the same as the one in UTXO model. Maybe can be put to common part.

Reply:
It could be done, but actually we do not have any common base class before MinimalState, which is in scorex. For the time being I did not any modification.